docs/release-notes/february-2024.md

```diff
@@ -0,0 +1,181 @@
+---
+slug: february-2024
+hide_title: true
+description: >-
+    Release notes include the changes, fixes, and additions in specific versions of Semgrep.
+toc_max_heading_level: 3
+title: February 2024
+tags:
+  - Release notes
+---
+
+# Semgrep release notes for February 2024
+
+## 🔧 OSS Engine
+
+* The following versions of the OSS Engine were released in February 2024:
+  * [<i class="fas fa-external-link fa-xs"></i>1.59.1](https://github.com/semgrep/semgrep/releases/tag/v1.59.1)
+  * [<i class="fas fa-external-link fa-xs"></i>1.60.0](https://github.com/semgrep/semgrep/releases/tag/v1.60.0)
+  * [<i class="fas fa-external-link fa-xs"></i>1.60.1](https://github.com/semgrep/semgrep/releases/tag/v1.60.1)
+  * [<i class="fas fa-external-link fa-xs"></i>1.61.1](https://github.com/semgrep/semgrep/releases/tag/v1.61.1)
+  * [<i class="fas fa-external-link fa-xs"></i>1.62.0](https://github.com/semgrep/semgrep/releases/tag/v1.62.0)
+  * [<i class="fas fa-external-link fa-xs"></i>1.63.0](https://github.com/semgrep/semgrep/releases/tag/v1.63.0)
+
+## 🌐 Cloud Platform
+
+### Added
+
+- API: Added a `rule` object under `findings` with the following fields: 
```

Comment:
@s-santillan should we link out to the API docs here? https://semgrep.dev/api/v1/docs/#tag/Finding/operation/semgrep_app.core_exp.findings.handlers.issue.openapi_list_recent_issues

Reply:
Added!